broccoli/minify.js

```diff
@@ -0,0 +1,32 @@
+const Funnel = require('broccoli-funnel');
+const Uglify = require('broccoli-uglify-js');
+const path = require('path');
+
+module.exports = function _minify(tree, name) {
+  return new Funnel(new Uglify(tree, {
```

Comment:
Would you mind moving this `new Uglify` invocation to a separate variable declaration?  It makes it easier to parse below which options go to which thing...